app/components/landing.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="landing-container">
+  <h2>Thank you for connecting your GitHub!</h2>
+  <h3>Please complete the signup in order to:</h3>
+  <ul class="landing-lists">
+    <li>Use Features</li>
+    <li>Display yourself on the (members) page</li>
+  </ul>
+
+  <div class="landing-button">
+      <LinkTo @route="signup" class="button">Get Started</LinkTo>
+  </div>
```

Comment:
Can we follow BEM conventions for naming classes? 
Will become easier to maintain consistency in future

Reply:
Ah! right, I missed it. Will make the change.

Reply:
8780fca